design/upf-selection-design.md

```diff
@@ -0,0 +1,185 @@
+# UPF Selection Design for Multiple DNNs
+
+## 1. Introduction
+- **Objective:** Modify the SMF to support UPF selection based on DNN and Slice ID.
```

Comment:
We have 2 options

1. Each UPF supports more than 1 DNN OR
2. Each UPF supports only 1 DNN but at SMF we need list of UPFs.

Reply:
Can we choose second option?

Reply:
Thanks. Then it means changes only at SMF and no need to do any changes at UPF.